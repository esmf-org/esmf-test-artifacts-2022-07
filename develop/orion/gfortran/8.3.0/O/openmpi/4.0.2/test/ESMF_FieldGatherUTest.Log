build time -- 2022-06-19 02:30:42
20220619 024101.408 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.408 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220619 024101.408 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220619 024101.408 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220619 024101.408 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220619 024101.409 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET0 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab2
20220619 024101.409 INFO             PET0 ESMF library build date/time: "Jun 19 2022" "02:29:36"
20220619 024101.409 INFO             PET0 ESMF library build location : /work/noaa/da/mpotts/sandbox/gfortran_8.3.0_openmpi_O_develop
20220619 024101.409 INFO             PET0 ESMF_COMM                   : openmpi
20220619 024101.409 INFO             PET0 ESMF_MOAB                   : enabled
20220619 024101.409 INFO             PET0 ESMF_LAPACK                 : enabled
20220619 024101.409 INFO             PET0 ESMF_NETCDF                 : enabled
20220619 024101.409 INFO             PET0 ESMF_PNETCDF                : disabled
20220619 024101.409 INFO             PET0 ESMF_PIO                    : enabled
20220619 024101.409 INFO             PET0 ESMF_YAMLCPP                : enabled
20220619 024101.410 INFO             PET0  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220619 024101.410 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220619 024101.411 INFO             PET0  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220619 024101.411 INFO             PET0  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220619 024101.411 INFO             PET0  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220619 024101.412 INFO             PET0  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220619 024101.412 INFO             PET0  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220619 024101.412 INFO             PET0 Finalizing ESMF
20220619 024101.409 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220619 024101.409 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220619 024101.409 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220619 024101.409 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220619 024101.409 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET1 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab2
20220619 024101.409 INFO             PET1 ESMF library build date/time: "Jun 19 2022" "02:29:36"
20220619 024101.409 INFO             PET1 ESMF library build location : /work/noaa/da/mpotts/sandbox/gfortran_8.3.0_openmpi_O_develop
20220619 024101.409 INFO             PET1 ESMF_COMM                   : openmpi
20220619 024101.409 INFO             PET1 ESMF_MOAB                   : enabled
20220619 024101.409 INFO             PET1 ESMF_LAPACK                 : enabled
20220619 024101.409 INFO             PET1 ESMF_NETCDF                 : enabled
20220619 024101.409 INFO             PET1 ESMF_PNETCDF                : disabled
20220619 024101.409 INFO             PET1 ESMF_PIO                    : enabled
20220619 024101.409 INFO             PET1 ESMF_YAMLCPP                : enabled
20220619 024101.410 INFO             PET1  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220619 024101.410 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220619 024101.411 INFO             PET1  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220619 024101.411 INFO             PET1  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220619 024101.411 INFO             PET1  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220619 024101.412 INFO             PET1  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220619 024101.412 INFO             PET1  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220619 024101.412 INFO             PET1 Finalizing ESMF
20220619 024101.409 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220619 024101.409 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220619 024101.409 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220619 024101.409 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220619 024101.409 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET2 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab2
20220619 024101.409 INFO             PET2 ESMF library build date/time: "Jun 19 2022" "02:29:36"
20220619 024101.409 INFO             PET2 ESMF library build location : /work/noaa/da/mpotts/sandbox/gfortran_8.3.0_openmpi_O_develop
20220619 024101.409 INFO             PET2 ESMF_COMM                   : openmpi
20220619 024101.410 INFO             PET2 ESMF_MOAB                   : enabled
20220619 024101.410 INFO             PET2 ESMF_LAPACK                 : enabled
20220619 024101.410 INFO             PET2 ESMF_NETCDF                 : enabled
20220619 024101.410 INFO             PET2 ESMF_PNETCDF                : disabled
20220619 024101.410 INFO             PET2 ESMF_PIO                    : enabled
20220619 024101.410 INFO             PET2 ESMF_YAMLCPP                : enabled
20220619 024101.410 INFO             PET2  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220619 024101.410 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220619 024101.411 INFO             PET2  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220619 024101.411 INFO             PET2  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220619 024101.411 INFO             PET2  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220619 024101.412 INFO             PET2  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220619 024101.412 INFO             PET2  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220619 024101.412 INFO             PET2 Finalizing ESMF
20220619 024101.409 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220619 024101.409 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220619 024101.409 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220619 024101.409 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220619 024101.409 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220619 024101.409 INFO             PET3 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab2
20220619 024101.409 INFO             PET3 ESMF library build date/time: "Jun 19 2022" "02:29:36"
20220619 024101.409 INFO             PET3 ESMF library build location : /work/noaa/da/mpotts/sandbox/gfortran_8.3.0_openmpi_O_develop
20220619 024101.409 INFO             PET3 ESMF_COMM                   : openmpi
20220619 024101.409 INFO             PET3 ESMF_MOAB                   : enabled
20220619 024101.409 INFO             PET3 ESMF_LAPACK                 : enabled
20220619 024101.409 INFO             PET3 ESMF_NETCDF                 : enabled
20220619 024101.409 INFO             PET3 ESMF_PNETCDF                : disabled
20220619 024101.409 INFO             PET3 ESMF_PIO                    : enabled
20220619 024101.409 INFO             PET3 ESMF_YAMLCPP                : enabled
20220619 024101.410 INFO             PET3  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
20220619 024101.410 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220619 024101.411 INFO             PET3  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
20220619 024101.411 INFO             PET3  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
20220619 024101.411 INFO             PET3  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
20220619 024101.412 INFO             PET3  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
20220619 024101.412 INFO             PET3  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
20220619 024101.412 INFO             PET3 Finalizing ESMF
