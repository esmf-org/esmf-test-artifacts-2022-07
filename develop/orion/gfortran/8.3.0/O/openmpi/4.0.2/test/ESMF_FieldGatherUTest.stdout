build time -- 2022-07-13 02:32:56
[1657698157.409157] [Orion-03-20:277193:0]       ib_verbs.h:84   UCX  ERROR ibv_exp_query_device(mlx5_0) returned 38: No space left on device
[1657698157.409131] [Orion-03-20:277195:0]       ib_verbs.h:84   UCX  ERROR ibv_exp_query_device(mlx5_0) returned 38: No space left on device
[1657698157.413768] [Orion-03-20:277192:0]       ib_verbs.h:84   UCX  ERROR ibv_exp_query_device(mlx5_0) returned 38: No space left on device
[1657698157.413730] [Orion-03-20:277194:0]       ib_verbs.h:84   UCX  ERROR ibv_exp_query_device(mlx5_0) returned 38: No space left on device
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldGatherUTest.F90, line 58
  NUMBER_OF_PROCESSORS           4
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 1d test, ESMF_FieldGatherUTest.F90, line 72
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 2d test, ESMF_FieldGatherUTest.F90, line 80
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldGather 3d test, ESMF_FieldGatherUTest.F90, line 88
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PASS  FieldScatter 2d test, ESMF_FieldGatherUTest.F90, line 96
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  Ending Test, file ESMF_FieldGatherUTest.F90, line 99
  PET           1  Test Elapsed Time    5.4649999999999981       msec.
  PET           2  Test Elapsed Time    5.6650000000000036       msec.
  PET           0  Test Elapsed Time    8.2710000000000008       msec.
  PET           3  Test Elapsed Time    5.4350000000000236       msec.
